Open Journal Systems Standard.js

```diff
@@ -0,0 +1,130 @@
+{
+	"translatorID": "a5d5ca83-b975-4abe-86c9-d956d7b9c8fa",
+	"label": "Open Journal Systems Standard",
```

Comment:
What's the difference between this translators and the regular "Open Journal Systems" translator? Is it necessary that both exist?

Reply:
i set the priority to 100